bindings/python/src/cloudsmith_api/models/repository_geo_ip_country_code_rules.py

```diff
@@ -19,7 +19,7 @@
 from cloudsmith_api.configuration import Configuration
 
 
-class InlineResponse200CountryCode(object):
+class RepositoryGeoIpCountryCodeRules(object):
```

Comment:
Think this looks like an example of somewhere where we've manually touched up the contract/schema to make the generated names in the bindings more palatable. Might need to do something similar elsewhere here...?